h2/src/main/org/h2/value/ValueLobDb.java

```diff
@@ -179,7 +179,7 @@ private static String createTempLobFileName(DataHandler handler)
         if (path.isEmpty()) {
             path = SysProperties.PREFIX_TEMP_FILE;
         }
-        return FileUtils.createTempFile(path, Constants.SUFFIX_TEMP_FILE, true, true);
+        return FileUtils.createTempFile(path, Constants.SUFFIX_TEMP_FILE, false, true);
```

Comment:
I think at the moment, if we are in embedded mode, we hand this file directly to the client jdbc code, and assume that the client will close and delete the file. If the client is badly written, this can leak handles.

Personally, I think that is the client codes problem, and the client should be fixed, but just noting a possible reason why this code is this way.

Reply:
`ValueLobDb` uses `FileStore.autoDelete()` method, so `TempFileDeleter` used by `Database` or `SessionRemote` should delete them after garbage collection of `ValueLobDb` values anyway. I don't see a reason for secondary system watcher that waits for JVM shutdown procedure and leaks memory.

Reply:
Great